### DISCUSSION TITLE
How often code from `master` branch is sync'ed to GAS example project?

### DISCUSSION THREAD

**Discussion by yermulnik** [derekantrican/GAS-ICS-Sync#469] [General]
The README says «[If you want to use this, please copy the script from here](https://script.google.com/d/1BOk8MDLbLaHh6SwG1M1tsgNXjkcC-79LE0QoipRuTDxbO3fMVvqoROQD/edit?newcopy=true)»?
Given some cool new feature or a long awaited fix has been merged into `master` branch, when it's expected to get propagated to the example `GAS-ICS-Sync` project so that consumers can copy files over to their own GAS projects?
Or is there a more reliable and streamlined way to pick up changes?
Thanks.

**Comment by jonas0b1011001** [comment]
The link in the readme will always point to the latest [release](https://github.com/derekantrican/GAS-ICS-Sync/releases).

You can make a copy of the current master branch from [here](https://script.google.com/home/projects/1B5pYZ1n7zwY7mW7qWiIr0KXA8Q86P5WbjzABjz_wr38pOVmwC_ld3Srj/edit) or update the files [manually](https://github.com/derekantrican/GAS-ICS-Sync/wiki/Setting-up-the-script-manually).

**Reply by yermulnik** [reply]
> You can make a copy of the current master branch from [here](https://script.google.com/home/projects/1B5pYZ1n7zwY7mW7qWiIr0KXA8Q86P5WbjzABjz_wr38pOVmwC_ld3Srj/edit) 

That's what I was asking about. I noticed the https://github.com/derekantrican/GAS-ICS-Sync/pull/467 had been merged, though the change wasn't at https://script.google.com/d/1BOk8MDLbLaHh6SwG1M1tsgNXjkcC-79LE0QoipRuTDxbO3fMVvqoROQD/edit?newcopy=true (which linked in README) yet and hence I wanted to know how often releases are sync'ed to GAS (the one which is linked in README and which I guess consumers use to create their GAS off of (as suggetsed by README) or update every so often to pick up changes, fixes, and improvements). Thanks.

**Reply by yermulnik** [reply]
So to re-cap: is there a way to get to know that the https://script.google.com/d/1BOk8MDLbLaHh6SwG1M1tsgNXjkcC-79LE0QoipRuTDxbO3fMVvqoROQD/edit?newcopy=true (which is linked in README) has updated to pickup changes from `master`?

**Reply by yermulnik** [reply]
Ah, so I guess I figured this:
* The «[If you want to use this, please copy the script from here](https://script.google.com/d/1BOk8MDLbLaHh6SwG1M1tsgNXjkcC-79LE0QoipRuTDxbO3fMVvqoROQD/edit?newcopy=true)» in README leads to incorrect (outdated) version of GAS
* The https://github.com/derekantrican/GAS-ICS-Sync/wiki/Setting-up-the-script-manually is outdated as is missing [`filters.gs`](https://github.com/derekantrican/GAS-ICS-Sync/blob/master/filters.gs)

These two evidentially were why I ended up with failed GAS script when attempted to update files manually 😕

**Reply by yermulnik** [reply]
* https://github.com/derekantrican/GAS-ICS-Sync/issues/470
  * https://github.com/derekantrican/GAS-ICS-Sync/pull/471
